.eslintrc.js

```diff
@@ -8,7 +8,7 @@ module.exports = {
   parserOptions: {
     ecmaVersion: 11,
   },
-  plugins: ['prettier'],
```

Comment:
The "Prettier" plugin for ESLint should not be removed -- we are using this to auto-format our code and fix common linting errors, such as missing semi-colons. I'd revert this change.